config/system-stats-monitor.json

```diff
@@ -0,0 +1,19 @@
+{
+	"disk": {
+		"metricsConfigs": {
+			"disk/io_time": {
+				"displayName": "disk/io_time"
+			},
+			"disk/weighted_io": {
+				"displayName": "disk/weighted_io"
+			},
+			"disk/avg_queue_len": {
+				"displayName": "disk/avg_queue_len"
+			}
+		},
+		"includeRootBlk": true,
+		"includeAllAttachedBlk": true,
+		"lsblkTimeout": "5s"
+	},
+	"invokeInterval": "60s"
```

Comment:
Do you expect all collectors in system stats monitor should use the same invoke interval?

Reply:
For now, yes, for simplicity.
If in future we want to allow different "sub system stats monitors" to use different interval, we could easily add another config option to allow it.